chain/cosmos/cosmos_chain.go

```diff
@@ -108,12 +108,12 @@ func (c *CosmosChain) getRelayerNode() *ChainNode {
 
 // Implements Chain interface
 func (c *CosmosChain) GetRPCAddress() string {
-	return fmt.Sprintf("http://%s:26657", c.getRelayerNode().Name())
+	return fmt.Sprintf("http://%s:26657", dockerutil.CondenseHostName(c.getRelayerNode().Name()))
```

Comment:
I think we might keep running into this when we want the hostname. I'll put up a PR of a suggestion.